Controllers/DashboardController.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace api.Controllers
 {
-    [Route("api/[controller]")]
+    [Route("api")]
```

Comment:
Not supposed to be like that, to be discussed :)

Reply:
Updated.